index.js

```diff
@@ -58,15 +58,18 @@ module.exports = {
       'always'
     ],
     'strict': 2,
+    'vars-on-top': 2,
```

Comment:
this is only valid for `var` afaik. So I changed it from warning to error so we don't do stupid shit with var in STB :)